src/util/tubemap.js

```diff
@@ -2315,6 +2315,13 @@ function calculateTrackWidth() {
 
 
 function getColorSet(colorSetName) {
+  // could be a color object from react-color
+  if(typeof colorSetName == "object") {
```

Comment:
Oh I guess you are using an object or a string as the way you determine whether you want to use a hex code or a defined palette?

I suppose you could use whether the string starts with "#" instead, though, right?